balta/src/test/resources/db/postgres/01_db.ddl

```diff
@@ -0,0 +1,6 @@
+CREATE DATABASE mag_db
```

Comment:
why `mag_` ? :D

Reply:
Well you had to notice, didn't you. 😉 
Well, I sense there are parts that might become common to _balta_, _fa-db_, _ultet_ (🤭 ) justifying a shared basic lib (mag = seed in Hu)

Reply:
curiosity is my superpower :D

Reply:
Okay, makes sense!